setup.cfg

```diff
@@ -0,0 +1,5 @@
+[flake8]
+hang-closing = true
+max-line-length = 120
+; E251 unexpected spaces around keyword / parameter equals
```

Comment:
Given we start from scratch, we can get rid of this ignored error, and follow PEP8 about the "no space around kwargs `=` operators` rule